### PR TITLE
update Gaia with to latest SDK master

### PR DESCRIPTION
**Pull request by fedekunze**
**Description**

Updated Gaia with the simulation refactors from https://github.com/cosmos/cosmos-sdk/pull/4869

______

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [x] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [x] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: app/utils.go [hunk 1]
```diff
@@ -1,20 +1,36 @@
...
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/x/staking"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
 )
 
+// SimulationOperations retrieves the simulation params from the provided file path
+// and returns all the modules weighted operations
+func SimulationOperations(app *GaiaApp, cdc *codec.Codec, config simulation.Config) []simulation.WeightedOperation {
```
Comment (fedekunze) [inline app/utils.go:14]: `simapp.SimulationOperations` receives a `SimApp` as parameter, so I can't use that util func atm. I'd need to extend `Simapp` to return the simulation manager (`app.sm`) and use a general `App` interface.

Comment (fedekunze) [reply app/utils.go:14]: I'd like to do that in a separate PR on the SDK tho

**Review by alessio — APPROVED** [review]
ACK - and ACK on https://github.com/cosmos/gaia/pull/210#discussion_r354015949

Please merge

File: app/utils.go (hunk 1 above)
Comment (fedekunze) [reply app/utils.go:14]: will open an issue on SDK

**Comment by jackzampolin** [comment]
All sims passing! 🎉🕺

**Pull request merged**